pkg/kubecost/mock.go

```diff
@@ -0,0 +1,667 @@
+package kubecost
+
+import (
+	"fmt"
+	"time"
+)
+const gb = 1024 * 1024 * 1024
+const day = 24 * time.Hour
+var disk = PVKey{}
+
+// NewUnitAllocation creates an *Allocation with all of its float64 values set to 1 and generic properties if not provided in arg
+func NewUnitAllocation(name string, start time.Time, resolution time.Duration, props *AllocationProperties) *Allocation {
```

Comment:
@Sean-Holcomb how do you feel about this name, now that it's exported from the package? Do you think it's clear enough that it's a test function, or do you think we should try to add the word "Mock" or "Test" in there to make it clear?

I'm imagining someone importing and using the `kubecost` package finding `kubecost.NewUnitAllocation(...)` and not understanding that it's a test-only function.

Reply:
I think that is a good point, I will go over them